### PR TITLE
Improve instructions in readme

### PR DESCRIPTION
**Pull request by aredridel**

### CODE REVIEW

**Comment by joshbedo** [comment]
+1 i just tried to clone the old link and got repository not found :)

File: README.md [hunk 1]
```diff
@@ -7,15 +7,20 @@ Kraken with Shopping Cart and PayPal integration
...
+
+Clone, install and run.
 
 ```shell
-$ git clone git@github.com:krakenjs/kraken-examples.git
-$ cd kraken-examples/with.shoppingcart
-$ npm install
-$ npm start
+git clone git@github.com:krakenjs/kraken-example-with-shoppingcart.git
+cd kraken-examples-with-shoppingcart
```
Comment (grawk) [inline README.md]: typo

Comment (aredridel) [inline README.md]: Fixed.

**Comment by grawk** [comment]
:+1:

**Pull request merged**